.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: Use this template for reporting bugs or issues.
+title: "[DATE] - Title"
+---
+# Bug Report
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/
+-->
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce this bug (as minimally and precisely as possible)**:
+
+**Anything else relevant for this bug report?**:
+
+**Environment**:
+
+- Kubernetes version (use `kubectl version`), please list client and server:
+- Controller version (provide the release tag or commit hash):
+- Cloud provider or hardware configuration:
```

Comment:
Add a line for Provisioner Name and Version

Reply:
Added